etc/taskcluster/decision_task.py

```diff
@@ -614,7 +614,7 @@ def wpt_chunks(platform, make_chunk_task, build_task, total_chunks, processes,
         # https://github.com/servo/servo/issues/22438
         if this_chunk == 0:
             task.with_script("""
-                time python2 ./mach test-wpt --release --binary-arg=--multiprocess \
+                time python ./mach test-wpt --release --binary-arg=--multiprocess \
```

Comment:
`time ./mach` (relying on shebang, like in other uses of `mach` commands) should also work, but this is fine.